staff/groups/aleti/project-app/src/logic/MovieApi.js

```diff
@@ -0,0 +1,39 @@
+const MovieApi = {
```

Comment:
BTW movieApi does not seem to be hiding the detail of being an API, instead, may "moviesLogic" fit better (the implementation details should not be exposed... now it can be an API, but tomorrow in could be another thing, so just "moviesLogic" works fine)

Reply:
+1!